src/backend/columnar/columnar_customscan.c

```diff
@@ -473,7 +497,7 @@ RecostColumnarSeqPath(RelOptInfo *rel, Oid relationId, Path *path)
 	 */
 	int numberOfColumnsRead = RelationIdGetNumberOfAttributes(relationId);
 	path->total_cost = path->startup_cost +
-					   ColumnarScanCost(rel, relationId, numberOfColumnsRead);
+					   ColumnarScanCost(rel, relationId, numberOfColumnsRead, 0);
```

Comment:
maybe have comment as we do for `int numberOfColumnsRead`:

```c
	/*
	 * Seq scan doesn't support projection pushdown, so we will read all the
	 * columns.
	 * Also note that seq scan doesn't support chunk group filtering too but
	 * our costing model already doesn't consider chunk group filtering.
	 */
	int numberOfColumnsRead = RelationIdGetNumberOfAttributes(relationId);

	/* seq scan cannot make use of any clauses */
	int numberOfClauses = 0;
	path->total_cost = path->startup_cost +
					   ColumnarScanCost(rel, relationId, numberOfColumnsRead, numberOfClauses);
```

Reply:
I moved the logic into CostColumnarSeqScan() and added a comment.